modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,344 @@
+
+| Update Documents
+| N/A
+| JSON that contains the following structure: { "matched" : int, "modified" : int, upsertedId: JSON }
+
+| Remove Documents
+| Query: JSON Object
+| N/A
+
+| Count Documents
+| Query: JSON Object
+| N/A
+
+| Find Documents
+| Query: JSON Object
+| Resolved dynamically based on the selected value of the collection parameter. The connector uses the structure of the first document in the given collection.
+*Reviwers, in the last sentence, what does the connector use the structure of the first document for?*
+
+| Create File
+| N/A
+a|** A GridFS JSON object with the following structure:
+* JSON Id
+* String fileName
+* Long Length
+* Int ChunkSize
+* Datetime uploadDate
+* JSON Metadata
+
+| Find Files
+| N/A
+a|** List<InputStream> +
+Each InputStream is a GridFS JSON object with the following attributes:
+* JSON Id
+* String fileName
+* Long Length
+* Int ChunkSize
+* Datetime uploadDate
+* JSON Metadata
+|===
+
+== Removed Operations
+
+The following operations were removed from the MongoDB connector:
+
+[%header,cols="15%,35%,15%,35%"]
+|===
+2+|Removed from MongoDB 5.x 2+| Can be reproduced in MongoDB 6.0.0 through
+|Name	| Description	|Name	|Description
+|Incremental dump | Executes an incremental dump of the database. | Dump | Use the Dump operation.
+|Restore | Takes the output from the dump and restores it | Restore from file or Restore from directory | These operations take the output from the dump file or directory and restore it.
+|Update documents by function | Update documents using a Mongo function | N/A | N/A
+|Update documents by functions | Update documents using one or more Mongo functions | N/A | N/A
+|Find one and update document | Finds and updates the first document that matches a given query | N/A | This operation's functionality can be reproduced by combining other operations
+*Reviewers, can you provide examples*
+|Save document | Inserts or updates a document based on its object ID | N/A | N/A
+|Find one document | Finds the first document that matches a given query | Find documents | Set the Limit=1 to return one document.
+|Execute generic command | Executes a generic command on the database | Execute command | Executes a command on the database
+|List files | Lists all files that match the given query and sorts them by filename | N/A | N/A
+|Find one file | Returns the first file that matches the given query | Find files | Giving a specific file to the query
+|===
+
+== Changes in Input Sources
+MongoDB 6.0.0 has one input source only, Object Listener, which retrieves all of the created documents that belong to a specific collection.
+
+The Delete Sources and Update Sources input sources were removed.
++
+*Reviewers, I don't see these sources. The only sources I see in the Reference Guide are Deleted Object, Modified Object, and New Object. Do you mean that these sources were removed?*
```

Comment:
Yes, Deleted Object and Modified Object were removed. New Object was replaced by Object Listener.